src/openrct2/object/ResourceTable.h

```diff
@@ -14,6 +14,7 @@
 #include <optional>
 #include <string>
 #include <vector>
+#include <cstdint>
```

Comment:
Please keep these sorted

```suggestion
#include <cstdint>
#include <optional>
#include <string>
#include <vector>
```